src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/IssuedTokenParametersEndpointAddressElement.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+
+namespace CoreWCF.Configuration
+{
+    internal sealed class IssuedTokenParametersEndpointAddressElement : EndpointAddressElementBase
+    {
```

Comment:
This was public on .NET Framework, was there a reason to make this internal?

Reply:
I made this Internal because it was not fully implemented.  Nor have tests been written for it.

Reply:
set to public